### PR TITLE
Open external links in new tab - Closes #523

### LINKED ISSUE #523: Update external links in SDK docs
Parent issue #521

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #523

### CODE REVIEW

File: modules/ROOT/pages/guides/frontend.adoc [hunk 1]
```diff
@@ -17,23 +17,23 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-* {url_angular}[Angular]
-* {url_react}[React]
-* {url_vue}[Vue]
+* {url_angular}[Angular^]
+* {url_react}[React^]
+* {url_vue}[Vue^]
 
-NOTE: This guide builds a frontend based on the {url_react_app}[Create React App] guide to create a frontend application.
+NOTE: This guide builds a frontend based on the {url_react_app}[Create React App^] guide to create a frontend application.
```
Comment (chris529) [inline modules/ROOT/pages/guides/frontend.adoc]:
```suggestion
NOTE: This guide covers how to build a frontend based on the {url_react_app}[Create React App^] guide to create a frontend application.
```

File: modules/ROOT/pages/guides/test.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 When the logic inside the custom transaction types becomes more complex, it becomes complicated to verify that the custom transaction logic is working as expected.
 
-Therefore it is recommended to write {url_unit_tests}[unit tests], that verify the logic of the transaction type.
+Therefore it is recommended to write {url_unit_tests}[unit tests^], that verify the logic of the transaction type.
```
Comment (chris529) [inline modules/ROOT/pages/guides/test.adoc:16]:
```suggestion
Therefore it is recommended to write {url_unit_tests}[unit tests^], that verifies the logic of the transaction type.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -142,7 +143,7 @@ All examples provided in the tutorials describe how to implement simple, but val
 
 The tutorials overview page provides an informative overview about all existing tutorials, including the estimated time and the skill level required to complete each specific tutorial.
 
-TIP: All code for the example apps that is used in the tutorials, is also available in the {url_github_sdk_examples}[lisk-sdk-examples repository] on GitHub.
+TIP: All code for the example apps that is used in the tutorials, is also available in the {url_github_sdk_examples}[lisk-sdk-examples repository^] on GitHub.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
TIP: All code for the example apps that is used in the tutorials is also available in the {url_github_sdk_examples}[lisk-sdk-examples repository^] on GitHub.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 4]
```diff
@@ -387,7 +385,7 @@ class HelloTransaction extends BaseTransaction {
 module.exports = HelloTransaction;
 ----
 
-TIP: Pleas see the file on Github: {url_github_hello_tx}[hello_world/hello_transaction.js]
+TIP: Pleas see the file on Github: {url_github_hello_tx}[hello_world/hello_transaction.js^]
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
TIP: Please see the following file on Github: {url_github_hello_tx}[hello_world/hello_transaction.js^]
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 5]
```diff
@@ -434,7 +432,7 @@ app <8>
     });
 ----
 
-TIP: Please see the file on Github: {url_github_hello_index}[hello_world/index.js].
+TIP: Please see the file on Github: {url_github_hello_index}[hello_world/index.js^].
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/hello-world.adoc:432]:
```suggestion
TIP: Please see the following file on Github: {url_github_hello_index}[hello_world/index.js^].
```

File: modules/ROOT/pages/tutorials/transport0.adoc [hunk 6]
```diff
@@ -124,12 +124,12 @@ Sensor wiring::
 image:lisk_rpi_ldr_and_temperature_sensors_wiring.png[RPI Wiring,title="RPI Wiring"]
 
 Preparing the Operating System::
-The operating system for the Raspberry needs to be copied from a computer into an SD card for this we recommend using {url_etcher} (good tutorial in {url_medium_pi_backup})
+The operating system for the Raspberry needs to be copied from a computer into an SD card for this we recommend using {url_etcher}[etcher^] (good tutorial at {url_medium_pi_backup}[medium^])
 +
-For preparing the Raspberry for `ssh` and wifi access follow this guide: {url_tutorial_pi}
+For preparing the Raspberry for `ssh` and wifi access follow this guide: {url_tutorial_pi}[pi tutorial^].
```
Comment (chris529) [inline modules/ROOT/pages/tutorials/transport0.adoc:129]:
```suggestion
For preparing the Raspberry for `ssh` and wifi access, please follow this guide: {url_tutorial_pi}[pi tutorial^].
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Looks very good...just a few minor points :-)

File: modules/ROOT/pages/guides/test.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/test.adoc:16]: I think it is correct like it is

File: modules/ROOT/pages/tutorials/transport0.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/tutorials/transport0.adoc:129]: I am not in favor of using `please` in docs a lot.

File: modules/ROOT/pages/tutorials/hello-world.adoc (hunk 5 above)
Comment (Tschakki) [reply modules/ROOT/pages/tutorials/hello-world.adoc:432]: We are referring to the code snippet above, so the `following` file sounds irritating for me here.

Comment (chris529) [reply modules/ROOT/pages/tutorials/hello-world.adoc:432]: OK, sounds good then yes in that case let's remove 'following.

File: modules/ROOT/pages/tutorials/transport0.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/tutorials/transport0.adoc:129]: OK, that's fine..

**Review by chris529 — COMMENTED** [review]
Looks very good :-)

**Review by chris529 — APPROVED** [review]
Done..

**Pull request merged**